home-manager/core.nix

```diff
@@ -12,6 +12,7 @@ in
   };
 
   config = {
+    services.getty.autologinUser = "core";
```

Comment:
Is there a particular reason why this has been (re-)enabled? That is, does it affect the bootup process? The downside of having this enabled is that it effectively allows root access on physical input -- plugging in a USB keyboard to an RPi or the computer has direct shell access, whereas without, you must do a remote login with the provided SSH keys.

Reply:
This does not have affect on booting the Raspberry Pi. Enabled it to have physical access because user password hasn't been set. IDK why I didn't just use remote login, will revert this.